src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts

```diff
@@ -82,15 +85,23 @@ export function useCreateSarcophagus() {
     archaeologistSignatures,
     archaeologistShards,
     arweaveTxIds,
-    currentStage,
+    currentStage
   });
 
+  // generates a random key with which to encrypt the outer layer of the sarcophagus
+  useCallback(async () => {
```

Comment:
Calling this out for scrutiny during review, I wasn't sure if this was the way to handle key generation and want to make sure it receives some attention

Reply:
The private key that encrypts the outer layer is the recipient's private key and is generated during the "Set Recipient" step. Are we changing this?

Reply:
inner layer should be the recipient private key, this only impacts the key the embalmer generates which gets distributed to archaeologists in shares